tests/ui/asm/parse-error.stderr

```diff
@@ -193,16 +193,12 @@ error: asm template must be a string literal
    |
 LL |         asm!(format!("{{{}}}", 0), in(reg) foo);
    |              ^^^^^^^^^^^^^^^^^^^^
-   |
-   = note: this error originates in the macro `format` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Does `-Zmacro-backtrace` still work for builtin-like macros with this PR?

Reply:
Yes, I've added two tests for it.